web/menu.jspf

```diff
@@ -117,7 +121,11 @@ org.opensolaris.opengrok.web.Util"
         if (ph.getGroups().size() > 0 && ph.getAllUngrouped().size() > 0) {
             %><optgroup label="Other"><%
         }
-        for ( Project p : ph.getAllUngrouped()) {
+        for (Project p : ph.getAllUngrouped()) {
```

Comment:
The latest change I did was to modify `cacheProjects()` to filter not-indexed projects so the changes in the cycle are probably not needed anymore.